processes/security_team_offboarding.md

```diff
@@ -0,0 +1,23 @@
+# Off-boarding Existing Security WG Members
+
+This document aims to provide a clear process for off-boarding existing WG members due to inactivity or any other eligible reason.
+
+## Process
+
+Every P period of time the WG will run through a process of validating membership relevancy as follows:
+
+* An issue will be posted on the WG Repository that will mention all existing WG members to query whether participation in the WG is still relevant.
+* After 2 weeks a reminder notice will be posted
+* After 4 weeks those who haven't responded to the issue will be placed in a pending status ('emeritus')
+* After 4 weeks those who haven't responded to the issue will be removed from the WG (see `Revoking Access to Confidential Systems` section for details on access removal check-list)
+
+At any time until the last time mark any member can chime in and request to retain his membership without requiring consensus reaching by the WG.
+
+## Revoking Access to Confidential Systems
+
+The following is a check-list for which existing WG members should be removed access from:
+* [ ] Remove user from [Repo README](https://github.com/nodejs/security-wg/blob/master/README.md), [Triage Team](https://github.com/nodejs/security-wg/blob/master/processes/third_party_vuln_process.md), [Security Team List](https://github.com/nodejs/security-wg/blob/master/processes/security_team_members.md), [GitHub CodeOwners](https://github.com/nodejs/security-wg/blob/master/.github/CODEOWNERS), 
```

Comment:
We can have an emeritus section

Reply:
Where do you think it's best to add it?

Reply:
Either at the end of the README, either in another doc. wdyt?

Reply:
to begin with we can probably just document it in this doc that discusses 'offboarding' to not juggle through too many doc files

Reply:
LGTM!